Yosemite/Yosemite/Stores/AnnouncementsStore.swift

```diff
@@ -0,0 +1,108 @@
+import Networking
+import Storage
+
+/// Protocol for `AnnouncementsRemote` mainly used for mocking.
+///
+public protocol AnnouncementsRemoteProtocol {
+
+    func getFeatures(appId: String,
```

Comment:
Thanks for thinking about unit testing. 
I think we should make this protocol to include `getAnnouncements...` so it matches 1-1 what is trying to hide(AnnouncementsRemote)